src/store/actions.js

```diff
@@ -998,8 +998,8 @@ export default {
 
 			try {
 				await setEnvelopeFlags(envelope.databaseId, {
-					$junk: !oldState,
-					$notjunk: oldState,
```

Comment:
https://github.com/nextcloud/mail/blob/1d5e5b49c1485e9be70f22c4df9526805f5238a7/src/store/actions.js#L991, https://github.com/nextcloud/mail/blob/1d5e5b49c1485e9be70f22c4df9526805f5238a7/src/store/actions.js#L1011-L1021 and similar stay?

Reply:
Yes, The UI handles the "$junk" but the backend compares the it as "junk", so in the UI they can all stay as "$junk" but need to have the $ sign stripped of when sent to the backend. The second image of allowed flags is comparison table, the first image is how they are compared/found.

Reply:
Sure, Its actual the line you highlighted,

Before:
```
await setEnvelopeFlags(envelope.databaseId, {
	$junk: !oldState,
	$notjunk: oldState,
})
```
After:
```
await setEnvelopeFlags(envelope.databaseId, {
	junk: !oldState,
	notjunk: oldState,
})
```